drape/drape_tests/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 project(drape_tests)
+include(FindPackageOrElseAddSubdirectory)
```

Comment:
Does it make sense to include this macro once from `cmake/OmimHelpers.cmake` and avoid inserting include into each 3p wrapper?

Reply:
Makes things simpler for sure. The original idea was to avoid silently including macros, as it unintentionally invoked macros are rather hard and unpleasant to debug. But now I see, that it is considered premature optimization here.  
So I moved the macro to `cmake/OmimHelpers.cmake`